src/document.ts

```diff
@@ -144,20 +155,14 @@ export class Document {
     );
   }
 
-  /**
-   * Returns the filename of the template to render.
-   */
-  get view(): string {
-    if (!this.fields) {
-      return (
-        (this.collection && this.collection.fields['$view']) || DEFAULT_VIEW
-      );
+  get view() {
+    if (this.fields && this.fields['$view']) {
+      return this.fields['$view'];
     }
-    return (
-      this.fields['$view'] ||
-      (this.collection && this.collection.fields['$view']) ||
-      DEFAULT_VIEW
-    );
+    if (this.collection && this.collection.fields['$view']) {
```

Comment:
In typescript you can simplify this to be `if (this.collection?.fields['$view']) {` to not need the extra check for the collection.